<?xml version="1.0"?>
<launch>

	<!-- start data acquisition -->
	<node pkg="cob_env_model" type="data_acquisition" name="data_acquisition" ns="data_acquisition" cwd="node" respawn="false" output="screen">
		<remap from="right/image_color" to="prosilica/right/image_raw"/>
		<remap from="image_xyz" to="sr4000/image_xyz"/>
		<remap from="image_grey" to="sr4000/image_grey"/>
	</node>
	
</launch>